pilot/helpers/Project.py

```diff
@@ -226,7 +226,7 @@ def get_files(self, files):
                 _, full_path = self.get_full_file_path(file_path, file_path)
                 file_data = get_file_contents(full_path, self.root_path)
             except ValueError:
-                file_data = {"path": file_path, "content": ''}
+                file_data = {"path": file_path, "name": os.path.basename(file_path), "content": ''}
```

Comment:
`name` is required by `Project.save_file()` so some edge cases had this failing.